### ISSUE TITLE
Update "Exact simulation with Qiskit primitives" to use V2 Primitives

### ISSUE THREAD

**Issue by kevinsung** [Qiskit/documentation#1072]
### Feedback

i.e. StatevectorEstimator and StatevectorSampler

https://docs.quantum.ibm.com/verify/simulate-with-qiskit-primitives

**Comment by javabster** [comment]
closing as duplicate of #1025

**Comment by kevinsung** [comment]
Actually, this can be done Aer primitives is released since it doesn't use them (it uses the reference primitives in Qiskit itself).

**Comment by kevinsung** [comment]
I'd like to first convert the page from MDX to Jupyter notebook, but this is blocked by https://github.com/Qiskit/documentation/pull/812 which modifies the page.
